sharing/index.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2017-01-26 23:40:06
+title: Sharing GIS Data
+categories: []
+---
+
+#### Quick Links
+
+[Submitting Data to the SGID]({% link sharing/process/index.md %}) — [Basic Metadata Creation Process]({% link sharing/metadata/index.md %}) — [AGRC Sharing Policies]({% link about/policy/index.md %})
+
+## The Joys of Sharing
+
+The Utah [State Geographic Information Database (SGID)]({% link sgid/index.html%}) is one of the premier statewide GIS databases in the country, providing clean and consistent authoritative data across every geography in the state. People have been turning to the SGID for nearly 30 years to support their personal, commercial, and government projects. Our webpages have had over 30,000 views and 5,000 downloads, and the (what database?) gets over 15,000 table views a month. This would not be possible without the help and collaboration of cities, counties, and agencies across the state.
+
+Many of our most popular datasets—[roads]({% link sharing/core-data/index.md %}#road_centerlines), [parcels]({% link sharing/core-data/index.md %}#parcels), [address points]({% link sharing/core-data/index.md %}#address_points)—depend on the cities and counties, who send us updated data on a regular basis. We then merge them into unified, state-wide datasets available to anyone, free of charge. Most counties also participate in the [Land Information Record]({% link sharing/core-data/index.md %}#UtahLIRParcels) project, providing detailed parcel information that is critical to coherent and coordinated regional planning and analysis.
+
+Many state agencies have taken advantage of the SGID's broad audience to share important data, too. Many individuals, business, and municipalities use [natural hazard layers]({% link data/geoscience/index.html %}) from the Utah Geological Survey and [land ownership boundaries]({% link data/cadastre/land-ownership/index.html %}) from the Public Lands Policy Coordination Office for site selection and analysis.
```

Comment:
**thought**: aren't land ownership boundaries from sitla not plpco?

Reply:
Ah, right, I was conflating the two in my head. Thanks!